### PR TITLE
Scala governance page proposal

### PR DESCRIPTION
**Pull request by romanowski**

### CODE REVIEW

File: proposals/030-governance-page.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+# The governance page for Scala
+
+## Proposer
+
+Proposed by Krzysztof Romanowski, VirtusLab, April 25th 2023
+
+## Abstract
+
+Scala as a language and its ecosystem are maintained, managed, and developed by various individuals and organizations. The teams and responsibilities are changing over time and, in general, get confusing the users.
```
Comment (ckipp01) [inline proposals/030-governance-page.md:13]:
```suggestion
Scala as a language and its ecosystem are maintained, managed, and developed by various individuals and organizations. The teams and responsibilities are changing over time and, which can cause confusion for the community.
```

File: proposals/030-governance-page.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Proposed by Krzysztof Romanowski, VirtusLab, April 25th 2023
+
+## Abstract
+
+Scala as a language and its ecosystem are maintained, managed, and developed by various individuals and organizations. The teams and responsibilities are changing over time and, in general, get confusing the users. 
+
+As an open-source community, Scala should be built on transparency, yet currently, it may not be clear who is making decisions or maintaining given parts of the ecosystem.
+
+This proposal is inspired by [Rust Governance](https://www.rust-lang.org/governance) page.
```
Comment (ckipp01) [inline proposals/030-governance-page.md:17]:
```suggestion
This proposal is inspired by the [Rust Governance](https://www.rust-lang.org/governance) page.
```

File: proposals/030-governance-page.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+
+## Proposal
+
+We propose to create a governance page listing:
+- brief description and links to all decision-making processes like SIP
+- teams maintaining the language and core tools (build tools, IDEs, and similar), including key positions
+- key initiatives and working groups (like Scala tooling summit)
```
Comment (ckipp01) [inline proposals/030-governance-page.md:25]:
```suggestion
- key initiatives and working groups (like the Scala tooling summit)
```

File: proposals/030-governance-page.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+## Cost
+
+There is an upfront cost to design and implement the page that may vary, deepening how detailed the page would be. It would involve a substantial amount of communication and coordination, but we believe this is aligned with what Scala Center is already doing. 
+
+Maintenance of the governance page should be similar to maintaining [SIP](https://github.com/scala/improvement-proposals) and [advisory board](https://github.com/scalacenter/advisoryboard) repositories.
+
+
+## Timescales
+
+Given the amount of communication and coordination involved, we expect that creating such a page may take a few months. Given that we finally have ScalaDays this year, Seattle or Madrid edition may be a good deadline for the first version of the page.
```
Comment (ckipp01) [inline proposals/030-governance-page.md:44]:
```suggestion
Given the amount of communication and coordination involved, we expect that creating such a page may take a few months. Given that we finally have ScalaDays this year, the Seattle or Madrid edition may be a good deadline for the first version of the page.
```

**Review by ckipp01 — COMMENTED** [review]
Thanks for your proposal @romanowski! I've gone ahead and added it to the agenda for the upcoming meeting. I added a few grammar nits.

**Comment by SethTisue** [comment]
currently the closest thing we have to this is https://scala-lang.org/community/#whos-behind-scala

once the page exists, we should link it from there

**Review by ckipp01 — APPROVED** [review]

**Pull request merged**